starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -1674,13 +1674,24 @@ sub check_star_catalog {
 		else { push @yellow_warn, $warn }
 	    }
 
-	    # Star within search box + search error and within 1.0 mags 
-	    if ($type ne 'MON' and $dz < $halfw + $search_err and $dy < $halfw + $search_err and $dm > -1.0) {
+            # Spoiler star in track box ACA-022
+            if (($type =~ /BOT|GUI/)
+                    and ($dz < $self->{dither_guide}->{ampl_p} + 25)
```

Comment:
I don't think that dither should be added in here since the two stars move in lock step.  Imagine a 64x64 dither and a spoiler at 40 arcsec away.  That is definitely fine.

Reply:
Oh good point.  In a dither habit.